src/borg/helpers.py

```diff
@@ -249,7 +255,7 @@ def write(self):
         self.tam_verified = True
         data = self.key.pack_and_authenticate_metadata(manifest.as_dict())
         self.id = self.key.id_hash(data)
-        self.repository.put(self.MANIFEST_ID, self.key.encrypt(Chunk(data, compression={'name': 'none'})))
+        self.repository.put(self.MANIFEST_ID, self.key.encrypt(Chunk(data), none_compression=True))
```

Comment:
I'd leave it with the existing compression support, that way no key.py changes are needed at all and no extra API.